File: 30 Days of Code/Python/Day 6 - Let's Review.py
# Enter your code here. Read input from STDIN. Print output to STDOUT

t = int(input())
for i in range(t):
    s = input()
    o = ''
    e = ''
    for j in range(len(s)):
        if j%2==0:
            e = e + s[j]
        else:
            o = o + s[j]
    print(e,o)

File: 30 Days of Code/PHP/Day 6 - Let's Review.php
<?php
$_fp = fopen("php://stdin", "r");
/* Enter your code here. Read input from STDIN. Print output to STDOUT */
$t = (int)readline();
for($i=0;$i<$t;$i++){
    $s = readline();
    $e = '';
    $o = '';
    for($j=0;$j<strlen($s);$j++){
        if($j%2==0){
            $e = $e.$s[$j];
        }
        else{
            $o = $o.$s[$j];
        }
    }
    echo($e." ".$o."\n");
}
?>